server/runtime/src/main/server/bin/server.sh

```diff
@@ -11,52 +11,43 @@ DIRNAME=`dirname "$0"`
 . "$DIRNAME/common.sh"
 
 while true; do
-   if [ "x$LAUNCH_ISPN_IN_BACKGROUND" = "x" ]; then
-      # Execute the JVM in the foreground
-      eval \"$JAVA\" $JAVA_OPTS \
-         -Dvisualvm.display.name=$PROCESS_NAME \
-         -Dinfinispan.server.home.path=\""$ISPN_HOME"\" \
-         -classpath \""$CLASSPATH"\" "$LOADER_CLASS" "$MAIN_CLASS" "$ARGUMENTS"
-      ISPN_STATUS=$?
-   else
-      # Execute the JVM in the background
-      eval \"$JAVA\" $JAVA_OPTS \
-         -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager \
-         -Dinfinispan.server.home.path=\""$ISPN_HOME"\" \
-         -classpath \""$CLASSPATH"\" "$LOADER_CLASS" "$MAIN_CLASS" "$ARGUMENTS" "&"
-      ISPN_PID=$!
-      # Trap common signals and relay them to the server process
-      trap "kill -HUP  $ISPN_PID" HUP
-      trap "kill -TERM $ISPN_PID" INT
-      trap "kill -QUIT $ISPN_PID" QUIT
-      trap "kill -PIPE $ISPN_PID" PIPE
-      trap "kill -TERM $ISPN_PID" TERM
-      if [ "x$ISPN_PIDFILE" != "x" ]; then
-        echo $ISPN_PID > $ISPN_PIDFILE
+   # Execute the JVM in the background
+   eval \"$JAVA\" $JAVA_OPTS \
+      -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager \
```

Comment:
You have lost the `visualvm.display.name`

Reply:
It seems it was never set when launching in the background. Updated :+1: